### PR TITLE
VIH-6768 add test PM and Winger redirected to video web

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-6768

### Change description ###

test for PM and Winger should be redirected to video-web

**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
[ ] No
```

### CODE REVIEW

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Login.feature [hunk 1]
```diff
@@ -43,4 +43,16 @@ Scenario: Representative completed checklist and self test redirected to Video W
...
+	Then the user is redirected to Video Web
+
+@VIH-4577 @PanelMember
+Scenario: Panel member redirected to Video Web
+	Given a new browser is open for a Panel Member
+	When the user logs in with valid credentials
+	Then the user is redirected to Video Web
+
+@VIH-4577 @Winger
+Scenario:  Winger redirected to Video Web
```
Comment (RuwanikaWeerakkody) [inline ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Login.feature:55]: @valentinabennett  could you add a test here to make sure JOH user role gets re-directed to v.web as well. That way it covers any other hearings roles in the future that have the JOH user role

**Review by npoxon — DISMISSED** [review]

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Login.feature (hunk 1 above)
Comment (npoxon) [reply ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Login.feature:55]: Thats going to be difficult from here though @RuwanikaWeerakkody, and the UserRole of JOH is on the conference, and the test setup will only know their user type (winger, panel member, individual etc...)

Comment (valentinabennett) [reply ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Login.feature:55]: we have user-type, I don't think I could do it with user role ( winger required special case type)

Comment (valentinabennett) [reply ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Login.feature:55]: user-role we get by user-type, currently we could not do by user-role

**Review by RuwanikaWeerakkody — APPROVED** [review]

**Review by npoxon — APPROVED** [review]

**Pull request merged**